packages/builder/src/components/rounds/Show.tsx

```diff
@@ -17,6 +17,100 @@ import Button, { ButtonVariants } from "../base/Button";
 import ErrorModal from "../base/ErrorModal";
 import LoadingSpinner from "../base/LoadingSpinner";
 import SwitchNetworkModal from "../base/SwitchNetworkModal";
+import { GrantsMetadataState } from "../../reducers/grantsMetadata";
+
+interface ApplyButtonProps {
+  roundState: any;
```

Comment:
@boudra can we update this from 'any' to the actual type?

Reply:
resolved!